src/main/java/com/blogify/blogapi/model/Comment.java

```diff
@@ -0,0 +1,39 @@
+package com.blogify.blogapi.model;
+
```

Comment:
By convention all DAO model is in /repository/model all model outside of /repository is used for DTO but in our case we use api generator and not create new model